ErrorStates[] data
ErrorStates[] ground
ErrorStates[] max_limits 
ErrorStates[] min_limits 
ErrorStates[] max_dist
ErrorStates[] min_dist
string folder


